plugins/CoreAdminHome/Emails/AnonymousAccessEnabledEmail.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Piwik;
+use Piwik\Plugins\CoreAdminHome\Emails\SecurityNotificationEmail;
+
+class AnonymousAccessEnabledEmail extends SecurityNotificationEmail
+{
+    /**
+     * @var string
+     */
+    private $siteName;
+
+    public function __construct($login, $emailAddress, $siteName)
+    {
+        $this->siteName = html_entity_decode($siteName);
```

Comment:
Can we perhaps add a comment why we need to decode the entities? Are they encoded back in the email mechanism itself?

Reply:
```suggestion
        // this property is used in a twig template, where it is escaped automatically
        // so we decode it here, to avoid double encoding
        $this->siteName = html_entity_decode($siteName);
```